srcpkgs/shadow/patches/fix-undefined-reference.patch

```diff
@@ -0,0 +1,19 @@
+--- a/lib/alloc.h
++++ b/lib/alloc.h
+@@ -84,14 +84,14 @@
+ }
+ 
+ 
+-inline void *
++static inline void *
```

Comment:
I don't like this patch.  You shouldn't need to make the functions static.  Please report an upstream bug so we can see what's broken.